apps/synapse-oauth-signin/src/test/App.test.tsx

```diff
@@ -34,11 +38,12 @@ const mockConsentToOAuth2Request = vi.mocked(
 const { ACCESS_TOKEN_COOKIE_KEY, POST_SSO_REDIRECT_URL_LOCALSTORAGE_KEY } =
   SynapseConstants
 function createParams(prompt?: string) {
+  const state: OAuth2State = { registrationUsername: '1ga1bi0qm' }
   const params = new URLSearchParams()
   params.set('response_type', 'code')
   params.set('client_id', '1234')
   params.set('scope', 'openid')
-  params.set('state', '1ga1bi0qm')
```

Comment:
I think this was originally meaningless (not a registration username), but the test broke when parsing since it's no longer an object